swap/src/cli.rs

```diff
@@ -78,6 +81,12 @@ pub enum Command {
         #[structopt(short = "id", long = "swap-id")]
         swap_id: Uuid,
 
+        #[structopt(short = "p", long = "connect-peer-id")]
```

Comment:
❗️ Please remove the option `p2p-address` (line 109) below in favor of these options. As far as I can see this is duplicated now. Note that for convenience it would help to keep  the `default` value, potentially also for the `alice_addr` above.

Please add another comment similar to the one on option `p2p-address` - the connection options are actually only relevant for Bob (there is a mistake in the current comment) because he has to be able to dial Alice. 
Since `Resume` is used by both parties this is a bit confusing but we can deal with it later.

Reply:
Done in https://github.com/comit-network/xmr-btc-swap/pull/102